common/tools/dev-tool/src/commands/run/testNodeTSInput.ts

```diff
@@ -24,9 +24,7 @@ export default leafCommand(commandInfo, async (options) => {
   const isModuleProj = await isModuleProject();
   const reporterArgs =
     "--reporter ../../../common/tools/mocha-multi-reporter.js --reporter-option output=test-results.xml";
-  const defaultMochaArgs = `${
-    isModuleProj ? "--loader=ts-node/esm " : "-r esm "
-  }-r ts-node/register ${reporterArgs} --full-trace`;
+  const defaultMochaArgs = `--loader=ts-node -r ts-node/register ${reporterArgs} --full-trace`;
```

Comment:
tell me why this is wrong - keep me honest. Pretend I know nothing (since I don't).

I tested this on a few packages and all worked as expected.